packages/client/graph-scaffolder/src/core/renderer.ts

```diff
@@ -324,6 +326,14 @@ export abstract class Renderer<V, E> extends EventEmitter {
 				.style('pointer-events', 'none');
 		}
 
+		window.addEventListener('keydown', (event) => {
+			if (event.key === 'z') this.isZoomable = true;
+		});
+
+		window.addEventListener('keyup', (event) => {
+			if (event.key === 'z') this.isZoomable = false;
+		});
```

Comment:
- I know we discussed this as a possible solution, but please check with the design team or Pascale if this is the desirable behaviour

- Rather than hardwiring `z` and use a isZoomable flag. You may want to consider having a zoom-modifier key as a parameter. If the modifier key is null or "" than everything is normal, if the modifier key is specified than you need to hold that key to zoom

- Rather than latching on to windows, you might be able to chec for the modifier keys in the zoom event and do early exit. This will be a bit cleaner, since windows is a global object, and also note here you never cleaned up things in here. 

- Treat graph-scaffolder as a library, if you change the default everyone suffers. So ideally we would just pass in the intentioned behaviour rather than forcing everyone else to opt-out.